warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1592,7 +1596,7 @@ static long getNextTime(long time, int granularity) {
      * @throws ParseException
      *             if the value failed to be parsed using the supplied format
      */
-    protected static long getTime(Object value, DateFormat format) throws ParseException {
+    static long getTime(Object value, DateFormat format) throws ParseException {
```

Comment:
I finally got to getting this built with our internal project.  This failed because this function is no longer available by an extending class.  This needs to be protected or public, but not package private.  This comment is the same for the methods below which you changed.  Thanks.